apps/backoffice-v2/src/pages/Entity/components/CallToAction/hooks/useCallToActionLogic/useCallToActionLogic.tsx

```diff
@@ -110,6 +110,10 @@ export const useCallToActionLogic = () => {
     workflow?.workflowDefinition?.config?.workflowLevelResolution ??
     workflow?.context?.entity?.type === 'business';
 
+  useEffect(() => {
```

Comment:
Fixes long-standing bug of an empty dropdown when changing the reject action while still sending a wrong rejection reason.